DigitalLearningSolutions.Web/Styles/viewComponents/textInput.scss

```diff
@@ -0,0 +1,8 @@
+.nhsuk-error-message.validation-errors-container__message {
+  margin-bottom: 4px;
+}
+
+.validation-errors-container {
+  padding-top: 4px;
+  padding-bottom: 16px;
+}

```

Comment:
Do you think any of the nhsuk-spacing values would work here? https://service-manual.nhs.uk/design-system/styles/spacing

e.g.
margin-bottom: nhsuk-spacing(1);